src/test/java/core/basesyntax/LambdaCalculatorTest.java

```diff
@@ -0,0 +1,161 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class LambdaCalculatorTest {
+    private static LambdaCalculator calculator;
+    private static final int TWENTY = 20;
+    private static final int FIFTEEN = 15;
+    private static final int ONE_HUNDRED = 100;
+    private static final int FIFTY = 50;
+    private static final int ZERO = 0;
+    private static final int FIFTY_SIX_THOUSANDS = 56000;
+    private static final int ADDITION_MAX_VALUE = -2;
+    private static final int SUBTRACTION_MAX_VALUE = 0;
+    private static final int DIVISION_MAX_VALUE = 1;
+    private static final int MULTIPLICATION_MAX_VALUE = 1;
+    private static final char ADDITION = '+';
+    private static final char SUBTRACTION = '-';
+    private static final char DIVISION = '/';
+    private static final char MULTIPLICATION = '*';
+    private static final char RAISING_TO_A_POWER = '^';
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new LambdaCalculator();
+    }
+
+    @Test
+    void testForAddition() {
+        double firstActual = calculator.calculate(Integer.MAX_VALUE, Integer.MAX_VALUE, ADDITION);
+        double secondActual = calculator.calculate(TWENTY, FIFTEEN, ADDITION);
+        double thirdActual = calculator.calculate(-ONE_HUNDRED, FIFTY, ADDITION);
+        double secondExpected = TWENTY + FIFTEEN;
```

Comment:
```suggestion
        double secondExpected = 35.0;
```
and so on
in tests this is ok